config/merge.go

```diff
@@ -54,6 +62,14 @@ func Merge(existingServices *ServiceConfigs, environmentLookup EnvironmentLookup
 
 	adjustValues(serviceConfigs)
 
+	if options.Postprocess != nil {
+		var err error
+		serviceConfigs, err = options.Postprocess(serviceConfigs)
```

Comment:
Ok, let's not add things we don't need for now and wait for user input then :angel: and let the signature like this right now :wink: